client/my-sites/themes/helpers.js

```diff
@@ -161,3 +163,9 @@ export function interlaceThemes( wpComThemes, wpOrgThemes, searchTerm, isLastPag
 
 	return interlacedThemes;
 }
+
+export function getTierRouteParam() {
```

Comment:
It would be awesome if we could use the tiers fetched from the API here (mapped to this specific problem domain, tier_slug) so that we don't have to maintain this.

Reply:
I think it's way too early to rely (or wait) on an API. 🤔 

Happy to be disproven, but this should be considered a nice to have for next year. 😄

Reply:
>Happy to be disproven, but this should be considered a nice to have for next year. 😄

Sure, next year seems good to me.